hangman/hangman.py

```diff
@@ -0,0 +1,83 @@
+# Hangman game to improve English and/or German vocabulary.
+
```

Comment:
As there are more packages/files I need to download to play this game it is good to know in which file the game starts and is the "main" file

Reply:
Beschreibung wie Spiel gestartet wird, wurd dem Readme hinzugefügt.